lte/gateway/c/core/oai/tasks/mme_app/mme_app_state_converter.cpp

```diff
@@ -177,19 +178,19 @@ void MmeNasStateConverter::proto_to_guti_table(
     const google::protobuf::Map<std::string, unsigned long>& proto_map,
     obj_hash_table_uint64_t* guti_htbl) {
   for (auto const& kv : proto_map) {
-    mme_ue_s1ap_id_t mme_ue_id = kv.second;
-    guti_t* guti_p             = (guti_t*) calloc(1, sizeof(guti_t));
+    mme_ue_s1ap_id_t mme_ue_id   = kv.second;
+    std::unique_ptr<guti_t> guti = std::make_unique<guti_t>();
+    memset(guti.get(), 0, sizeof(guti_t));
 
-    mme_app_convert_string_to_guti(guti_p, kv.first);
+    mme_app_convert_string_to_guti(guti.get(), kv.first);
     hashtable_rc_t ht_rc = obj_hashtable_uint64_ts_insert(
-        guti_htbl, guti_p, sizeof(*guti_p), mme_ue_id);
+        guti_htbl, guti.get(), sizeof(guti.get()), mme_ue_id);
```

Comment:
`sizeof(guti.get())` should have been `sizeof(guti_t)`.

Reply:
no problem @electronjoe, mme restart tests are good ones to run for PRs when making changes to state managers, converters, etc.